#include "util/gui/flags.h"

#include "absl/log/log.h"

namespace gui {

#include "util/gui/color_edit_flags.inc"
#include "util/gui/input_text_flags.inc"
#include "util/gui/popup_flags.inc"
#include "util/gui/slider_flags.inc"
#include "util/gui/style_color.inc"
#include "util/gui/style_var.inc"
#include "util/gui/table_flags.inc"
#include "util/gui/window_flags.inc"

}  // namespace gui
